spec/system/work_show_spec.rb

```diff
@@ -16,4 +17,12 @@
     expect(page).to have_content "IS_CITED_BY"
     expect(page).to have_content "https://www.biorxiv.org/content/10.1101/545517v1"
   end
+
+  it "copies DOI to the clipboard" do
+    sign_in user
+    visit work_path(work)
+    expect(page).to have_content "COPY"
+    find("button#copy-doi").click
+    expect(page).to have_content "COPIED"
```

Comment:
I cannot get this change to be detected on the test. 

The "click" of the button triggers the JavaScript to change the label from "COPY" to "COPIED" and the test passes if I run it with `RUN_IN_BROWSER=true` but not without it.

Reply:
@carolyncole sadly that's not it - the spec in this case already has that setting (https://github.com/pulibrary/pdc_describe/pull/709/files#diff-4c1af586c2edeb41cfdbc5021707f0df6bafd14cecbedc8e52e596e3ab4eaa21R11) 

What is weird about this particular scenario is that the test passes OK if I run in the browser (I can see the old/new text, and the test actually passes), but continues to fails if I don't run it on the browser.

Reply:
Sorry!  I have defiantly had that same behavior in the past.  I just can not remember what caused it.

Reply:
I changed this test completely. I am now testing the presence of the HTML `button` tag rather than the JS call. I left a note in the test with the complications in case we later on find a better way to test it.